### DISCUSSION TITLE
java client read file error

### DISCUSSION THREAD

**Discussion by zyjnd** [seaweedfs/seaweedfs#5922] [Q&A]
I found,the http request headers have Authentication with jwt[token] when the java client write file,but it's don't have Authentication with jwt[token] when the java client read,so i can't read file by java client

**Comment by chrislusf** [comment]
java client did not implement jwt for read operations yet.

**Reply by zyjnd** [reply]
> java client did not implement jwt for read operations yet.

ok,thanks
